### ISSUE TITLE
CI failures on RC and Next for TypeScript 3.5.0

### ISSUE THREAD

**Issue by JoshuaKGoldberg** [palantir/tslint#4745]
Example Circle builds:

https://circleci.com/gh/JoshuaKGoldberg/tslint/1472
https://circleci.com/gh/JoshuaKGoldberg/tslint/1468

**Comment by ryanelian** [comment]
I am also experiencing this bug. It also affects tsutils used by tslint unfortunately.

```
npm WARN tslint@5.16.0 requires a peer of typescript@>=2.1.0 || >=2.1.0-dev || >=2.2.0-dev || >=2.3.0-dev || >=2.4.0-dev || >=2.5.0-dev || >=2.6.0-dev || >=2.7.0-dev || >=2.8.0-dev || >=2.9.0-dev || >=3.0.0-dev || >= 3.1.0-dev || >= 3.2.0-dev but none is installed. You must install peer dependencies yourself.
npm WARN tsutils@2.29.0 requires a peer of typescript@>=2.1.0 || >=2.1.0-dev || >=2.2.0-dev || >=2.3.0-dev || >=2.4.0-dev || >=2.5.0-dev || >=2.6.0-dev || >=2.7.0-dev || >=2.8.0-dev || >=2.9.0-dev || >= 3.0.0-dev || >= 3.1.0-dev but none is installed. You must install peer dependencies yourself.
```

```
=> Found "tsutils@2.29.0"
info Reasons this module exists
   - "tslint" depends on it
```

**Comment by adidahiya** [comment]
Fixed by #4751
